.github/workflows/cargo-deny.yaml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+      - '**'
+      - '!master'
+
+name: cargo-deny
+
+jobs:
+  deny-check:
+    name: cargo-deny
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: EmbarkStudios/cargo-deny-action@v1
```

Comment:
My only comment is that I'm unclear if Rust version matters for cargo-deny, if so we may want to specify, otherwise 👍

Reply:
I don't know how to tell whether Rust version matters?

I mimicked the examples on their homepage plus two examples of real users with workflows set up. That doesn't answer the question of whether the Rust version matters though.

Reply:
There's no options on that action to specify, so it's probably fine 🤷 ; if it's a problem we can deal with it later